test/remote/gateways/remote_adyen_test.rb

```diff
@@ -363,6 +363,30 @@ def test_successful_purchase_with_risk_data
     assert_equal '[capture-received]', response.message
   end
 
+  def test_successful_purchase_with_splits
+    split_data = [{
```

Comment:
I'm still feeling a bit concerned about this test and whether the format I'm using for this test is accurate, i.e., using strings instead of symbols.

Reply:
One way to possibly do this is to pin the AM version in your Gemfile to your local repo, then go through "Testing production-like calls". Happy to work through this maybe this afternoon?

Reply:
I've tested a local production-like purchase request through Core and Active Merchant, and the syntax seems to be working correctly! :tada: